gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -533,4 +533,9 @@ ChangeRequest getChangeRequest(String zoneName, String changeRequestId,
    * @see <a href="https://cloud.google.com/dns/api/v1/changes/list">Cloud DNS Chages: list</a>
    */
   Page<ChangeRequest> listChangeRequests(String zoneName, ChangeRequestListOption... options);
+
+  /**
+   * Initiates a new empty batch ready to be populated with service calls.
```

Comment:
Also, maybe instead of "ready to be populated with service calls" "for grouping multiple service calls in one underlying RPC call"?